inference-engine/src/mkldnn_plugin/nodes/mkldnn_topk_node.cpp

```diff
@@ -235,119 +1701,15 @@ void MKLDNNTopKNode::top1(const float* src_data, float* dst_data, int* dst_idx,
     });
 }
 
-template <class Compare1, template <typename> class Compare2>
-void MKLDNNTopKNode::topk_axis(const float* src_data, float* dst_data, int* dst_idx, SizeVector in_dims) {
+void MKLDNNTopKNode::topk_axis(const float* src_data, float* dst_data, int32_t* dst_idx, SizeVector in_dims, std::function<float(float, float)> compare) {
```

Comment:
SizeVector can be replaced with const SizeVector&

Reply:
Fixed. Thanks Egor!